common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -147,18 +183,27 @@ def get_html(self):
             'depends': ';'.join(self.required_html_ids)
         })
 
+    def author_view(self, context):
```

Comment:
This  has_author_view has value = true by default because it is identified in class https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/studio_editable.py#L13